tests/bwc/test_rolling_upgrade.py

```diff
@@ -148,8 +161,13 @@ def _test_rolling_upgrade(self, path, nodes):
             # Run a query as a user created on an older version (ensure user is read correctly from cluster state, auth works, etc)
             with connect(cluster.node().http_url, username='arthur', password='secret', error_trace=True) as custom_user_conn:
                 c = custom_user_conn.cursor()
-                wait_for_active_shards(c, expected_active_shards)
+                wait_for_active_shards(c)
```

Comment:
Behaviour change starting `5.6`. With `deny dql on table doc.t1 to arthur`, arthur cannot see `t1` anymore -> cannot wait for `t1` to be active anymore.